File: julia/primerParcial/problema17.jl
A = [120 -20 1;-80 80 1;-40 -60 120]
B = [400;0;200]
x = inv(A) * B
println("Las concentraciones son:")
println("C1= ",x[1])
println("C2= ",x[2])
println("C3= ",x[3])

File: python/primerParcial/problema17.py
import numpy as np
A = [[120, -20, 1], [-80, 80, 1], [-40, -60, 120]]
B = [[400],[0], [200]]
x = np.dot(np.linalg.inv(A), B)
print("Las concentraciones son:")
print("C1= "+ str(x[0]))
print("C2= "+ str(x[1]))
print("C3= "+ str(x[2]))